foundryconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dataPath": "D:/FoundryVTT",
```

Comment:
?

Reply:
Just some dirty i forgot to put on the .gitignore. No problem I remove it